config/v1/types_cluster_operator.go

```diff
@@ -124,6 +124,10 @@ const (
 	// The binary maintained by the operator (eg: openshift-apiserver for the openshift-apiserver-operator) may still be
 	// available, but the user intent cannot be fulfilled.
 	OperatorFailing ClusterStatusConditionType = "Failing"
+
+	// OperatorUpgradeable indicates whether or not the operator is in a state where it can accept a new payload.  When
```

Comment:
Don't use payload, that word is being scrubbed, and don't use `CVO`.  Use this description instead:

> Upgradeable indicates whether the operator is in a state that is safe to upgrade. When status is `False` administrators should not upgrade their cluster and the message field should contain a human readable description of what the administrator should do to allow the operator to successfully update.